src/core/Akka.Cluster/ClusterSettings.cs

```diff
@@ -66,7 +66,11 @@ public ClusterSettings(Config config, string systemName)
                 ) ? TimeSpan.Zero :
                 clusterConfig.GetTimeSpan("down-removal-margin", null);
 
+#pragma warning disable CS0618
             AutoDownUnreachableAfter = clusterConfig.GetTimeSpanWithOffSwitch("auto-down-unreachable-after");
+            if (AutoDownUnreachableAfter == default(TimeSpan)) // need to restore correct defaults now that it's been deleted from HOCON file
```

Comment:
This is a work-around needed to restore the "correct" defaults for `AutoDownUnreachableAfter` now that it's no longer included in the default `cluster.conf` file.